lib/pdk/module/templatedir.rb

```diff
@@ -236,26 +232,44 @@ def self.files_in_template(dirs)
       #
       # @api private
       def config_for(dest_path)
-        if @config.nil?
-          config_path = File.join(@path, 'config_defaults.yml')
+        sync_config_path = File.join(@path, '/.sync.yml')
+        config_path = File.join(@path, 'config_defaults.yml')
 
-          if File.file?(config_path) && File.readable?(config_path)
-            begin
-              @config = YAML.safe_load(File.read(config_path), [], [], true)
-            rescue StandardError => e
-              PDK.logger.warn(_("'%{file}' is not a valid YAML file: %{message}") % { file: config_path, message: e.message })
-              @config = {}
-            end
-          else
-            @config = {}
-          end
+        if @config.nil?
+          confdefaults = read_config(config_path)
```

Comment:
Tiny nitpick here, but just for consistency sake, `confdefaults` and `syncconfig` should be underscore separated like the other variables are.